clients/spark/core/src/main/scala/io/treeverse/clients/GarbageCollector.scala

```diff
@@ -446,4 +463,30 @@ object GarbageCollector {
 
     bulkRemove(df, MaxBulkSize, bucket, region, awsRetries, snPrefix, hcValues).toDF("addresses")
   }
+
+  private def writeParquetReport(
+      dstRoot: String,
+      df: DataFrame,
+      time: String,
+      suffix: String = ""
+  ) = {
+    val dstPath = dstRoot + s"/dt=${time}/${suffix}"
+    df.write.parquet(dstPath)
+  }
+
+  private def writeJsonSummary(
+      dstRoot: String,
+      numDeletedObjects: Long,
+      gcRules: String,
+      hcValues: Broadcast[ConfMap],
+      time: String
+  ) = {
+    val dstPath = new Path(dstRoot + s"/dt=${time}/summary.json")
```

Comment:
```suggestion
    val dstPath = new Path(s"${dstRoot}/dt=${time}/summary.json")
```

Also, I really like interpolating values into strings like this -- can we do it in the paths above, please, for consistency?